.github/workflows/ci-cd.yml

```
@@ -0,0 +1,102 @@
+    environment: prod
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+      
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v3
+      
+    - name: Build Docker image
+      run: |
+        docker buildx build --platform linux/amd64 -t bilbomusicbot:latest --load .
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)